src/library_sdl.js

```diff
@@ -2905,14 +2905,14 @@ var LibrarySDL = {
     return SDL.setGetVolume(SDL.music, volume);
   },
 
-  Mix_LoadMUS_RW__docs: '/** @param {number} a1 */',
-  Mix_LoadMUS_RW: 'Mix_LoadWAV_RW',
+  Mix_LoadMUS_RW__deps: ['Mix_LoadWAV_RW'],
+  Mix_LoadMUS_RW: (filename) => _Mix_LoadWAV_RW(filename, 0),
 
   Mix_LoadMUS__deps: ['Mix_LoadMUS_RW', 'SDL_RWFromFile', 'SDL_FreeRW'],
   Mix_LoadMUS__proxy: 'sync',
   Mix_LoadMUS: (filename) => {
     var rwops = _SDL_RWFromFile(filename, 0);
-    var result = _Mix_LoadMUS_RW(rwops, 0);
+    var result = _Mix_LoadMUS_RW(rwops);
```

Comment:
This change is need in order to prevent a closure error when running `other.test_closure_full_js_library_pthread`.

```
building:ERROR: /tmp/emscripten_temp_2b2wdh02/hello_world.js:37694:24: WARNING - [JSC_TYPE_MISMATCH] initializing variable
found   : function(?, number): ?
required: function(number): ?
  37694|   var _Mix_LoadMUS_RW = _Mix_LoadWAV_RW;
                                 ^^^^^^^^^^^^^^^

/tmp/emscripten_temp_2b2wdh02/hello_world.js:37707:19: WARNING - [JSC_WRONG_ARGUMENT_COUNT] Function _Mix_LoadMUS_RW: called with 2 argument(s). Function requires at least 1 argument(s) and no more than 1 argument(s).
  37707|       var result = _Mix_LoadMUS_RW(rwops, 0);
                            ^^^^^^^^^^^^^^^^^^^^^^^^^

0 error(s), 2 warning(s), 67.4% typed
```

I have no idea at all why this change would trigger that error since it seems like a clear pre-existing error.  Closure is such a mystery.